### ISSUE TITLE
Examples for trigger events such as onAdd, onDelete in the documentation

### ISSUE THREAD

**Issue by max-carroll** [Yuvaleros/material-ui-dropzone#207]
### Describe the problem related to this feature request

Add some examples for the events to the DropzoneArea.md, this will enable us to easily verify that we haven't broken it whilst developing new features and also showcase how to use the feature.

We could include

- onChange (although I think they already use this)
- onDelete
- onDrop
- onDropRejected

### Describe the solution you'd like

Perhaps we could even have one example which does all? Thoughts?

### Describe alternatives you've considered

We could stash a version of the DropzoneArea.md which we unstash when we need to test it
We could have something separate just for development which would make it easier to verify it

**Comment by mattcorner** [comment]
I'll drop my thoughts in :)

I do think the docs need some more use case examples. I'm not sure that one for each of this is that useful though. Theres a balance between helpful examples, and overload in the docs.

From a developer experience aspect, I absolutely agree on the need for an environment to bash and test these components. A sample react app (dev only) with multiple examples in for developers to bash test during development is one solution I've seen. Storybook etc are others. 

The best validation though is of course a test suite environment.

**Comment by max-carroll** [comment]
@max-carroll Yes I agree storybook would be a good solution for the component devs, then we wouldn't need lots of examples in the docs just to develop against
